packages/core/src/index.ts

```diff
@@ -43,6 +43,11 @@ interface PersistentStorage {
   set: (key: string, value: string) => void;
 }
 
+type flagCacheConfig = {
```

Comment:
types should be uppercase :)

```suggestion
type FlagCacheConfig = {
```